WordPress/Classes/ViewRelated/Blog/QuickStartChecklistManager.swift

```diff
@@ -63,7 +65,7 @@ extension QuickStartChecklistManager: UITableViewDelegate {
     }
 
     func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
-        let headerView = tableView.dequeueReusableHeaderFooterView(withIdentifier: QuickStartChecklistHeader.identifier) as? QuickStartChecklistHeader
+        let headerView = tableView.dequeueReusableHeaderFooterView(withIdentifier: QuickStartChecklistHeader.defaultReuseID) as? QuickStartChecklistHeader
```

Comment:
The registration code looks like this:

```
tableView.register(QuickStartChecklistHeader.defaultNib, forHeaderFooterViewReuseIdentifier: QuickStartChecklistHeader.defaultReuseID)
```

The `identifier` was coming from `Kanvas` and its `Identifiable` protocol.